modules/ROOT/pages/to-download-and-install-studio-wx.adoc

```diff
@@ -8,11 +8,33 @@ endif::[]
 . Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Download Site].
 . Extract the Windows installation file into the `C:\` root folder.
 ** Ensure that your user has read and write permissions for the `C:\` directory.
-** If you are using a Windows antivirus application, ensure that the `plugins/` and `features/` directories are under the _trusted_ category.
+** If you are using a Windows antivirus application, ensure that the `plugins/` and `features/` directories are under the *trusted* category.
 . Open the extracted Anypoint Studio.
 +
 When Studio first runs, the embedded OpenJDK Platform libraries might cause a Windows Security alert. +
 Ensure to allow the OpenJDK Platform library from Temurin publisher through your Windows firewall.
-. Click OK to accept the default workspace. +
+. Click *OK* to accept the default workspace. +
 ** If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or later. +
 Studio 7.x supports only Mule 4.x projects, because the structure of the project, export format, XML, and scripting language are different than the ones used by Mule 3.x.
+
+== Exclude Studio from Windows Defender Scans
+
+While Windows Defender is a tool for keeping your computer safe, it can also slow down the performance of Studio because it scans files in real-time as they are accessed, which causes delays. To prevent performance issues, you can completely exclude Studio from Windows Defender real-time protection.
+
+. Run Studio. The following Windows Defender real-time protection notice pop-ups to ask if Studio folders and processes should be excluded:
++
+image::windows-defender-exclusions.png[“Windows Defender real-time protection exclusions popup asks the user if folder and processes should be excluded.”]
++
+* */.p2* folder stores Studio plugins.
++
+* The second folder shows the path of the workspace the user selects when launching Studio.
++
+* The third folder point to Anypoint Studio application location.
++
+. Optionally, select the *Remember my decision* checkbox.
+. Click *Yes*.
+. Click *Yes* again when Windows PowerShell requests permission to make the change. 
++
+image::windows-powershell-access-request.png[“Windows Powershell popup requests user to confirm the change.”]
+
+NOTE: If you select the *Remember my decision* checkbox and click *No*, Windows Defender real-time protection scans Studio folders and processes as you work with it.
```

Comment:
"with it" is not clear. Does this mean as you work with Studio or as you work with Windows Defender? If the former - "as you work with them."

Reply:
Great catch! Thanks!